plugins/modules/web_app_pool.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2015, Henrik Wallström <henrik@wallstroms.nu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: web_app_pool
+short_description: Configure IIS Web Application Pools
+version_added: 1.0.0
```

Comment:
Because this is the initial release we don't need to say when it was added.

```suggestion
```

Reply:
Please convert the documentation file to a ymal file